ens-migration/technical-description.md

```diff
@@ -21,7 +21,7 @@ A new version of the ENS registry has been deployed, and can be found at address
 * Addition of \`setRecord\` and \`setSubnodeRecord\` methods, which allow setting owner, resolver, and TTL in a single operation.
 * Addition of an approval mechanism based on ERC721, that allows users to delegate control over their names to another address, without having to transfer their names.
 
-The new registry implementation has been [audited by Sam Sun](https://gist.github.com/samczsun/2f0a2e266191042baada48c5407d8986), as well as being reviewed by Consensys dilligence; no issues were found by either auditor.
+The new registry implementation has been [audited by Sam Sun](https://gist.github.com/samczsun/2f0a2e266191042baada48c5407d8986), as well as [reviewed by Consensys Dilligence](https://diligence.consensys.net/audits/2019/03/ens-permanent-registrar/); no issues were found by either auditor.
```

Comment:
Removing the word "being" is good. Spelling of ConsenSys and Diligence should be fixed (capital S, no double L). That audit report is for a different component, so that link should be dropped.